### DISCUSSION TITLE
Online cron specific URL in the documentation seems to be false

### DISCUSSION THREAD

**Discussion by Gaston1201** [FreshRSS/FreshRSS#5769] [Q&A]
**Describe the bug**
The documentation for [refreshing the feeds with online cron](https://freshrss.github.io/FreshRSS/en/users/09_refreshing_feeds.html#online-cron) seems to be false.

**To Reproduce**
1. Go to [https://freshrss.example.net/i/?c=feed&a=actualize](https://freshrss.example.net/i/?c=feed&a=actualize) (it could be different depending on your installation).
2. Received message : "Not Found, The requested URL was not found on this server".

**Environment information (please complete the following information):**
- Device: PC
- OS: Windows 10
- Browser: Firefox 119.0
- FreshRSS version: 1.21.0
- Database version: sqlite
- PHP version: 8.0.30
- Installation type: OVHcloud free web hosting 100M - Online cron with cron.job.org

**Additional context**
It seems that [https://freshrss.example.net/**p**/i/?c=feed&a=actualize](https://freshrss.example.net/p/i/?c=feed&a=actualize) (it could be different depending on your installation) is the correct URL. See the **/p** added to the URL.

**Comment by Frenzie** [comment]
It's recommended to use e.g. .htaccess to manipulate the URL that way. I'm on my phone atm so I can't quite immediately find it but it's in the deployment parts of the readme/manual.

**Comment by Alkarex** [comment]
> It seems that [https://freshrss.example.net/**p**/i/?c=feed&a=actualize](https://freshrss.example.net/p/i/?c=feed&a=actualize) (it could be different depending on your installation) is the correct URL. See the **/p** added to the URL.

No, this is a sign of wrong Web root in your Web server configuration.
Are you using Apache, nginx, something else?

Check https://freshrss.github.io/FreshRSS/en/admins/10_ServerConfig.html

In particular, for Apache:

https://github.com/FreshRSS/FreshRSS/blob/1aa43e894e08815ce153e681f8b119c21e878797/docs/en/admins/10_ServerConfig.md?plain=1#L26

or for nginx:

https://github.com/FreshRSS/FreshRSS/blob/1aa43e894e08815ce153e681f8b119c21e878797/docs/en/admins/10_ServerConfig.md?plain=1#L87

See also https://freshrss.github.io/FreshRSS/en/admins/03_Installation.html

**Reply by Gaston1201** [reply]
> No, this is a sign of wrong Web root in your Web server configuration.

My instance of FreshRSS has been uploaded (FTP) on a "basic" webserver (no database, no cron) provided by an hoster (OVHcloud). I'm afraid I cannot make some changes in the configuration.

**Reply by Alkarex** [reply]
You might have an option to change the Web root. If not, then it is a *sub folder* installation, which is also fine, although a bit less optimal. In the case of a sub-folder installation, the root of FreshRSS is `https://example.net/path/FreshRSS/p/` instead of `https://freshrss.example.net/` on a dedicated server. In your case, it looks like you have put the FreshRSS code at the root of your FTP folder (so there is no `path/FreshRSS/`), which is also fine.
